<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCWClass</class>
 <widget class="QMainWindow" name="PCWClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointCloudWare</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="widgetWin" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>450</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	border: 2px solid white;
	border-radius: 10px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(10,102,150), stop:1 rgb(0,0,0));
 }

</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/PCW/Resources/main/soft.png);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="actionOpenOne"/>
     <addaction name="actionOpenMany"/>
     <addaction name="action_Openmesh"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionRecentOpenFile"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_PointData">
    <property name="title">
     <string>激光雷达点信息</string>
    </property>
    <addaction name="actionLASData"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>窗口</string>
    </property>
    <widget class="QMenu" name="menu_Win">
     <property name="title">
      <string>新建3D窗口</string>
     </property>
     <addaction name="action_newVTK"/>
     <addaction name="action_newGL"/>
    </widget>
    <addaction name="menu_Win"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="action_CloseAll"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据管理</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>格式转换</string>
     </property>
     <addaction name="action_las2txt"/>
     <addaction name="action_las2pcd"/>
     <addaction name="action_txt2pcd"/>
     <addaction name="action_obj2pcd"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_PointData"/>
   <addaction name="menu"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenOne"/>
   <addaction name="actionOpenMany"/>
   <addaction name="action_Openmesh"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProperties">
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>工程</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTreeView" name="TableSet"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockConsole">
   <property name="windowTitle">
    <string>输出</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="Console">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenOne">
   <property name="icon">
    <iconset theme="打开单个点云" resource="pcw.qrc">
     <normaloff>:/PCW/Resources/main/open-alt.png</normaloff>:/PCW/Resources/main/open-alt.png</iconset>
   </property>
   <property name="text">
    <string>打开单个点云</string>
   </property>
   <property name="toolTip">
    <string>打开单个点云</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="保存文件" resource="pcw.qrc">
     <normaloff>:/PCW/Resources/main/save.png</normaloff>:/PCW/Resources/main/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="另存为文件" resource="pcw.qrc">
     <normaloff>:/PCW/Resources/main/save-as.png</normaloff>:/PCW/Resources/main/save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为文件</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionRecentOpenFile">
   <property name="icon">
    <iconset theme="最近打开的文件">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>最近打开的文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="退出">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionLASData">
   <property name="icon">
    <iconset theme="LAS点云数据信息">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>LAS点云数据信息</string>
   </property>
  </action>
  <action name="actionOpenMany">
   <property name="icon">
    <iconset theme="打开多个点云(同格式)" resource="pcw.qrc">
     <normaloff>:/PCW/Resources/main/many-open-folder.png</normaloff>:/PCW/Resources/main/many-open-folder.png</iconset>
   </property>
   <property name="text">
    <string>打开多个点云(同格式)</string>
   </property>
   <property name="toolTip">
    <string>打开多个点云(同格式)</string>
   </property>
  </action>
  <action name="action_newVTK">
   <property name="icon">
    <iconset theme="新建VTK视图">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>新建VTK视图</string>
   </property>
   <property name="toolTip">
    <string>新建VTK视图</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="关闭">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="action_newGL">
   <property name="icon">
    <iconset theme="新建GL视图">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>新建GL视图</string>
   </property>
   <property name="toolTip">
    <string>新建GL视图</string>
   </property>
  </action>
  <action name="action_las2txt">
   <property name="icon">
    <iconset theme="las转txt">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>las转txt</string>
   </property>
  </action>
  <action name="action_las2pcd">
   <property name="icon">
    <iconset theme="las转pcd">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>las转pcd</string>
   </property>
  </action>
  <action name="action_txt2pcd">
   <property name="icon">
    <iconset theme="txt转pcd">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>txt转pcd</string>
   </property>
  </action>
  <action name="action_obj2pcd">
   <property name="icon">
    <iconset theme="obj转pcd">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>obj转pcd</string>
   </property>
  </action>
  <action name="action_Openmesh">
   <property name="icon">
    <iconset resource="pcw.qrc">
     <normaloff>:/PCW/Resources/main/openmesh.png</normaloff>:/PCW/Resources/main/openmesh.png</iconset>
   </property>
   <property name="text">
    <string>打开网格化文件</string>
   </property>
   <property name="toolTip">
    <string>打开网格化文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pcw.qrc"/>
 </resources>
 <connections/>
</ui>
